bin/setup.sh

```diff
@@ -3,14 +3,22 @@ ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd .. && pwd)
 
 TITLE="WP-Docker-Construct"
 ADMIN_EMAIL="engenharia@log.pt"
-URL="docker-local.dev"
-SINGLE_SITE=1
 REPOSITORY="git@github.com:log-oscon/WP-Construct.git"
 
 echo "-----------------------------"
 echo "${TITLE}"
 echo "-----------------------------"
 
+read -p "Write your domain: " site_url
+URL="${site_url// /-}.loc";
```

Comment:
Isn't `.loc` a risky option?

Reply:
+1 for this.  Maybe .local?

Reply:
test?

Reply:
Damn need more coffee. Yeah, .test NOT .local

Reply:
I think `.local` is a better option, since `.test` is defined as the test ambient.  
You don't use that, but we shouldn't assume what other will do with this repo.

Reply:
I like .local better. But is it "safe" to use? I'm good either way so it's up to you guys.